pretender.js

```diff
@@ -333,13 +335,15 @@ Pretender.prototype = {
   passthrough: PASSTHROUGH,
   checkPassthrough: function checkPassthrough(request) {
     var verb = request.method.toUpperCase();
-
     var path = parseURL(request.url).fullpath;
-
-    verb = verb.toUpperCase();
```

Comment:
This was weird. Do you see a reason for that ?

Reply:
certainly an oversight